app/components/sdg/goals/target_component.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="tabs-panel <%= active %>" id="tab_<%= type %>_targets">
```

Comment:
IMHO the `tabs-panel` is easier to understand if written inside the `TargetsComponent`. This would make this component (potentially) more reusable, since it would be just a definition list which could be inserted anywhere.

The code in the `TargetsComponent` view could be something like:

```erb
<ul class="tabs" ...>
...
</ul>

<div class="tabs-content" data-tabs-content="target_tabs">
  <div class="tabs-panel ...">
    <%= render SDG::Goals::TargetComponent.new(global_targets) %>
  </div>
  <div class="tabs-panel ...">
    <%= render SDG::Goals::TargetComponent.new(local_targets) %>
  </div>
</div>
```

If you'd like to avoid the duplication this change introduces, we could do something like:

```erb
<% [global_targets, local_targets].each do |targets| %>
  <div class="tabs-panel ...">
    <%= render SDG::Goals::TargetComponent.new(targets) %>
  </div>
<% end %>
```

Let me know what you think :thinking:. One inconvenient could be the logic behind the `active` method.